src/Sylius/Behat/Context/Api/Shop/CheckoutContext.php

```diff
@@ -1176,6 +1218,18 @@ public function iShouldSeeWithUnitPrice(ProductInterface $product, int $unitPric
         Assert::true($this->hasProductWithUnitPrice($product->getName(), $unitPrice));
     }
 
+    /**
+     * @Given I use :type checkout type
+     */
+    public function iUseSpecificCheckoutType(string $type): void
+    {
+        if ($type === 'async') {
```

Comment:
type is not secured in any way, f.eg Given I use XD checkout type would lead to normal checkout.